### PR TITLE
Feature/206/landing page event section

### PR DESCRIPTION
**Pull request by aliceluo2048**
### Describe the problem being solved:
The next upcoming event would be displayed on the home page with RSVP button. If there are no events, a default message is shown with no RSVP buttons.

**After examples with an event and no event**
 
Screenshot from 2019-10-23 11-14-09
Screenshot from 2019-10-23 11-58-31

**Before - mobile  changes fixing the text within the buttons**

before-button
**After**

after-button

### Impacted areas in the application: 

List general components of the application that this PR will affect: 
* 

PR checklist
- [X] I included  a screenshot for FE changes
- [X] I have linked the PR to a Zenhub ticket
- [X] I have checked for merge conflicts
- [X] I have run the [prettier](https://prettier.io/) command `make pretty`

### CODE REVIEW

**Comment by aliceluo2048** [comment]
https://github.com/the-difference-engine/ymim/blob/440b2ea6b42487e06c0607f597a1490c2d6b999b/frontend/src/components/home/midsection/lower_mid.jsx#L24
Changing the findingNextUpcomingEvent utils function to return a boolean instead of the event object will affect EventList.js (under events component), where the function is called and returns the event object, which is passed as a prop. I haven't made changes to the utils function.

File: frontend/src/components/home/midsection/lower_mid.jsx [hunk 1]
```diff
@@ -1,62 +1,84 @@
...
 import footerImage4 from "../../../assets/chicago-illinois-skyline-skyscrapers-161963.jpeg";
 
 class LowerMid extends Component {
+  componentDidMount() {
+    this.props.fetchEvents();
+  }
+
   render() {
+    const defaultContent =
+      "Check back again soon for what's is coming up next for YMIM. If you have an event that you think YMIM should be a part of please please email: Founder@theymim.org or call: 773.941.1200";
```
Comment (mkarazin) [inline frontend/src/components/home/midsection/lower_mid.jsx]: Two quick thoughts:
- Strings should be placed in a separate config or strings file. This makes it easy to update, easy to localize (tho we never will), and will clean up this code.
- For performance reasons, you should not declare static variables within a render function (or any function for that matter). At our scale, this won't really affect much. But on a much bigger site, it starts adding up. Instead, declare them at a portion of the class or file which will only be instantiated once.

File: frontend/src/components/home/midsection/lower_mid.jsx [hunk 2]
```diff
@@ -1,62 +1,84 @@
...
+      <>
+        <h2 className="text-center heading">What's New?</h2>
+        {nextEvent ? (
+          <h2 className="text-center sub-heading">
+            <Moment format="llll">{nextEvent.start.local}</Moment>
+            {" - " + nextEvent.name.text}
+          </h2>
+        ) : (
+          ""
+        )}
```
Comment (mkarazin) [inline frontend/src/components/home/midsection/lower_mid.jsx]: Style choice, I'd recommend avoiding the ternary operator when one of the true/false clauses are empty. Instead use `&&`.

e.g.: `{ nextEvent && (<h2...../>) }`

File: frontend/src/components/home/midsection/lower_mid.jsx [hunk 3]
```diff
@@ -1,62 +1,84 @@
...
+                rel="noopener noreferrer"
+                className="button-anchors"
+                alt="Facebook"
+              >
+                Facebook
+              </a>
+            </button>
+            <button className="ym-button" id="enroll">
+              <a
+                href="https://www.eventbrite.com/o/young-masterbuilders-in-motion-inc-18024803549"
```
Comment (mkarazin) [inline frontend/src/components/home/midsection/lower_mid.jsx]: Ideally, this URL is stored in a config file and not in the presenter

**Pull request merged**